docs/proposals/20220927-label-sync-between-machine-and-nodes.md

```diff
@@ -90,14 +88,19 @@ As a cluster admin/user, for the purpose of workload placement, I would like a d
 
 As a cluster admin/user, I want that Cluster API label management on Kubernetes Nodes doesn't conflict with labels directly managed by users or by other controllers.
 
+
+### Story 4
+
+As a cluster admin/user, I would like a declarative and secure means by which to annotate my Cluster API-managed nodes.
```

Comment:
I read this more as feature than a story driven by a use case. Can we have a more specific story? Why would I as a cluster admin want to annotate the Node through capi machine api. Can you describe an service/API consumer use case where this feature provides value?

Reply:
e.g. for labels centralized cluster admin control to express node semantics to influence workload placement is good argument